### PR TITLE
ci: add sqldiff action

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: .github/workflows/rotki_sqldiff.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: SQLDiff
+
+on:
+  pull_request_target:
+    branches:
+      - main
+
```
Comment (yabirgb) [inline .github/workflows/rotki_sqldiff.yml:7]: is this correct? Shouldn't it be develop/bugfixes/master?

File: .github/workflows/rotki_sqldiff.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      - name: Install required
+        run: sudo apt-get install -y sqlcipher sqlite3-tools
+
+      - name: SQLDiff action
+        id: sql-diff
+        uses: rotki/action-sqldiff@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          files: |
+            *.db
```
Comment (yabirgb) [inline .github/workflows/rotki_sqldiff.yml:31]: the | is needed?

```suggestion
          files: *.db
```

Comment (kelsos) [reply .github/workflows/rotki_sqldiff.yml:31]: it's either that or quoting I guess

File: .github/workflows/rotki_sqldiff.yml (hunk 1 above)
Comment (kelsos) [reply .github/workflows/rotki_sqldiff.yml:7]: True

**Review by yabirgb — APPROVED** [review]
lgtm

**Pull request merged**